pkg/deployer/clusterapiserver.go

```diff
@@ -43,9 +43,9 @@ type caCertParams struct {
 	tlsKey   string
 }
 
-func getApiServerCerts() (*caCertParams, error) {
+func getApiServerCertsForNamespace(ns string) (*caCertParams, error) {
 	const name = "clusterapi"
-	const namespace = corev1.NamespaceDefault
+	namespace := ns
```

Comment:
Potentially just name the function argument `namespace` and don't introduce a new variable.

Reply:
Good suggestion!